.github/workflows/main.yml

```
@@ -207,11 +207,16 @@ jobs:
           dart test
   test_flutter:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        flutter: [3.19, 3.22]
```

Add a comment explaining why these, and when they should be updated.

```suggestion
        # We want to support the two latest stable Flutter versions
        flutter: [3.19, 3.22]
```